central/vulnmgmt/vulnerabilityrequest/manager/requestmgr/singleton.go

```diff
@@ -2,10 +2,12 @@ package requestmgr
 
 import (
 	imageCVEDataStore "github.com/stackrox/rox/central/cve/image/datastore"
+	imageCVE2DS "github.com/stackrox/rox/central/cve/image/v2/datastore"
 	deploymentDataStore "github.com/stackrox/rox/central/deployment/datastore"
 	imageDataStore "github.com/stackrox/rox/central/image/datastore"
 	"github.com/stackrox/rox/central/reprocessor"
 	"github.com/stackrox/rox/central/sensor/service/connection"
+	views "github.com/stackrox/rox/central/views/imagecveflat"
```

Comment:
Are you sure this is the one you need?  Doesn't `github.com/stackrox/rox/central/views/imagecves` give you everything you need?